matrix_client/api.py

```diff
@@ -179,6 +188,10 @@ def create_room(self, alias=None, is_public=False, invitees=None):
             content["room_alias_name"] = alias
         if invitees:
             content["invite"] = invitees
+        if name:
+            content["name"] = name
+        if federate is not None:
+            content["creation_content"] = {'m.federate': federate}
```

Comment:
Looking at area of spec linked by @Half-Shot above, where is the "creation_content" field coming from? Is this more unspecced synapse behavior?